src/main/java/de/samply/icd10dictionary/dao/IcdCodeDaoPostgres.java

```diff
@@ -39,6 +39,16 @@ public int insert(IcdCode icdCode) {
     return 0;
   }
 
+  @Override
+  public List<IcdCode> selectListOfIcdCodesByCode(String codeParam) {
```

Comment:
Please move this method under the selectIcdCodeByCode method, because you use this order in the interface.